identifiers/country-nz/parliament_electorates.csv

```diff
@@ -1,73 +1,82 @@
-ocd-division/country:nz/general_electorate:papakura,Papakura
-ocd-division/country:nz/general_electorate:port_hills,Port Hills
-ocd-division/country:nz/general_electorate:rangitata,Rangitata
-ocd-division/country:nz/general_electorate:rangitikei,Rangitikei
-ocd-division/country:nz/general_electorate:rimutaka,Rimutaka
-ocd-division/country:nz/general_electorate:rodney,Rodney
-ocd-division/country:nz/general_electorate:rongotai,Rongotai
-ocd-division/country:nz/general_electorate:rotorua,Rotorua
-ocd-division/country:nz/general_electorate:selwyn,Selwyn
-ocd-division/country:nz/general_electorate:tamaki,Tamaki
-ocd-division/country:nz/general_electorate:taranaki-king_country,Taranaki-King Country
-ocd-division/country:nz/general_electorate:taupo,Taupo
-ocd-division/country:nz/general_electorate:tauranga,Tauranga
-ocd-division/country:nz/general_electorate:te_atatu,Te Atatu
-ocd-division/country:nz/general_electorate:tukituki,Tukituki
-ocd-division/country:nz/general_electorate:upper_harbour,Upper Harbour
-ocd-division/country:nz/general_electorate:waikato,Waikato
-ocd-division/country:nz/general_electorate:waimakariri,Waimakariri
-ocd-division/country:nz/general_electorate:wairarapa,Wairarapa
-ocd-division/country:nz/general_electorate:waitaki,Waitaki
-ocd-division/country:nz/general_electorate:wellington_central,Wellington Central
-ocd-division/country:nz/general_electorate:west_coast-tasman,West Coast-Tasman
-ocd-division/country:nz/general_electorate:whanganui,Whanganui
-ocd-division/country:nz/general_electorate:whangarei,Whangarei
-ocd-division/country:nz/general_electorate:wigram,Wigram
-ocd-division/country:nz/maori_electorate:hauraki-waikato,Hauraki-Waikato
-ocd-division/country:nz/maori_electorate:ikaroa-rawhiti,Ikaroa-Rawhiti
-ocd-division/country:nz/maori_electorate:tamaki_makaurau,Tamaki Makaurau
-ocd-division/country:nz/maori_electorate:te_tai_hauauru,Te Tai Hauauru
-ocd-division/country:nz/maori_electorate:te_tai_tokerau,Te Tai Tokerau
-ocd-division/country:nz/maori_electorate:te_tai_tonga,Te Tai Tonga
-ocd-division/country:nz/maori_electorate:waiariki,Waiariki
+id,name,validFrom,validTo
+ocd-division/country:nz,New Zealand,,
+ocd-division/country:nz/general_electorate:auckland_central,Auckland Central,2017-11-07,
+ocd-division/country:nz/general_electorate:bay_of_plenty,Bay of Plenty,2017-11-07,2020-09-19
+ocd-division/country:nz/general_electorate:botany,Botany,2017-11-07,
+ocd-division/country:nz/general_electorate:christchurch_central,Christchurch Central,2017-11-07,
+ocd-division/country:nz/general_electorate:christchurch_east,Christchurch East,2017-11-07,
+ocd-division/country:nz/general_electorate:clutha-southland,Clutha-Southland,2017-11-07,2020-09-19
+ocd-division/country:nz/general_electorate:southland,Southland,2020-09-19,
+ocd-division/country:nz/general_electorate:coromandel,Coromandel,2017-11-07,
+ocd-division/country:nz/general_electorate:dunedin_north,Dunedin North,2017-11-07,2020-09-19

```

Comment:
I believe these and many others of the electoral districts were valid before 2017-11-07:
https://en.wikipedia.org/wiki/Dunedin_North_(New_Zealand_electorate)

validFrom should reflect either when the electoral districts were created or they can be left blank.

Reply:
Removed validFrom dates of existing electoral districts.